src/prommis/uky/uky_flowsheet.py

```diff
@@ -220,14 +218,25 @@ def main():
 
     scaled_model = set_scaling(m)
 
-    assert_units_consistent(scaled_model)
-    assert degrees_of_freedom(scaled_model) == 0
+    if degrees_of_freedom(scaled_model) != 0:
+        raise AssertionError(
+            "The degrees of freedom are not equal to 0."
+            "Check that the expected variables are fixed and unfixed."
+            "For more guidance, run assert_no_structural_warnings from the IDAES DiagnosticToolbox "
+        )
 
     initialize_system(scaled_model)
 
+    solve(scaled_model)
```

Comment:
I would rename this to `solve_system` to be consistent with `initialize_system`, and to avoid any confusion with the solver object.